Header header

#Current settings of the motors [-1.0,1.0]
#negative is reverse (if available), 0.0 stop, positive forward
float64 left
float64 right
